Reorder the default players, and pre-set the link to our mplayer executable
Also add an icon

diff --git src/players.cpp src/players.cpp
index 7291c8b..8e0547e 100644
--- src/players.cpp
+++ src/players.cpp
@@ -111,10 +111,10 @@ Players::Players() {
 	//list.push_back( Player("yoDownet", "yoDownet.exe", "%u -o %f", false, false, Player::VideoAudio) );
 	#endif
 #else
+	list.push_back( Player("MPlayer", "@@PREFIX@@/bin/mplayer", "%u -title %t", false, false, Player::Video) );
 	list.push_back( Player("SMPlayer", "smplayer", "%u", true, true, Player::Video) );
 	list.push_back( Player("SMPlayer (audio)", "smplayer", "%u -media-title %t", true, false, Player::Audio) );
 	list.push_back( Player("SMPlayer (add to playlist)", "smplayer", "-add-to-playlist %u", true, true, Player::VideoAudio) );
-	list.push_back( Player("MPlayer", "mplayer", "%u -title %t", false, false, Player::Video) );
 	list.push_back( Player("VLC", "vlc", "%u --meta-title=%t", false, true, Player::VideoAudio) );
 	list.push_back( Player("Dragon Player", "dragon", "%u", false, false, Player::VideoAudio) );
 	list.push_back( Player("Totem", "totem", "%u", false, false, Player::VideoAudio) );
diff --git src/smtube.pro src/smtube.pro
index 28bbdc0..4cc3bdd 100644
--- src/smtube.pro
+++ src/smtube.pro
@@ -88,6 +88,8 @@ unix {
 	DEFINES += TRANSLATION_PATH=$(TRANSLATION_PATH)
 }
 
+ICON = smtube.icns
+
 win32 {
 	RC_FILE = smtube.rc
 	CONFIG(debug, debug|release) {
